File: Instruções Sequenciais/Q09.cpp
/*
------------------------------------------------
9. Crie um algoritmo/fluxograma/programa que 
peça ao usuário para inserir dois números 
inteiros e, em seguida, imprima o resultado 
arredondado da divisão do primeiro pelo segundo.
Usar a biblioteca cmath.
 */
#include <iostream>
#include <cmath>
#include <locale.h>
using namespace std;
int main () {
	setlocale(LC_ALL,"Portuguese");
	int num1, num2;
	float resultado;
	cout<<"Informe a primeiro número:";
	cin>>num1;
	cout<<"Informe o segundo número:";
	cin>>num2;	
	resultado = round((float)num1 /(float)num2);
	cout<<"A divisão arredondada é: ";
	cout<<resultado<<endl;
	return 0;
}

File: exercicios/python/Instruções Sequenciais/Q09.py
# Exercício 9 — Divisão arredondada usando cmath
import math

def main():
    a = int(input("Informe o primeiro número inteiro: "))
    b = int(input("Informe o segundo número inteiro: "))
    resultado = math.round(a / b)
    print(f"Resultado da divisão arredondada ({a} / {b}): {resultado}")

if __name__ == "__main__":
    main()